components/admin_console/admin_user_card/admin_user_card.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {Client4} from 'mattermost-redux/client';
+
+import ProfilePicture from 'components/profile_picture';
+import * as Utils from 'utils/utils.jsx';
+
+import './admin_user_card.scss';
+
+type BulletProps = {
+    user: {
+        first_name: string | null;
+        last_name: string | null;
+        nickname: string | null;
+        last_picture_update?: number | undefined;
+    };
+}
+
+type Props = {
+    user: {
+        first_name: string | null;
+        last_name: string | null;
+        nickname: string | null;
+        last_picture_update?: number | undefined;
+        id: string;
+    };
+    body?: JSX.Element | undefined;
```

Comment:
`body` and `footer` can actually be typed `React.ReactNode`. That type just includes everything renderable in React, so it has `JSX.Element`, `undefined`, strings, and so on are all included.

Reply:
Hi @hmhealey  I already change it.